test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Please make sure that test.md is not presented in your repo and it ignored by github